pom.xml

```diff
@@ -92,7 +92,7 @@
     <top.dir>${project.basedir}/..</top.dir>
 
     <!-- Dependency versions -->
-    <jackson-bom.version>2.12.6.20220326</jackson-bom.version>
+    <jackson-bom.version>2.15.1</jackson-bom.version>
```

Comment:
@ksobolew - thanks for the contribution. We usually try to keep our dependencies in sync with HBase. Looks like they're currently on 2.14.1, so we should probably do the same, unless we find a particular incompatibility or CVE with that version. 

See https://github.com/apache/hbase/blob/branch-2.5/pom.xml#L578

Reply:
Sure, 2.14.1 is good too. Regarding CVEs, both 2.15.1 and 2.14.1 are clear of them, contrary to 2.12.6.

Reply:
Changed, thanks.